hudi-common/src/main/java/org/apache/hudi/common/table/timeline/HoodieDefaultTimeline.java

```diff
@@ -354,6 +354,11 @@ public Stream<HoodieInstant> getInstants() {
     return instants.stream();
   }
 
+  @Override
+  public List<HoodieInstant> getInstantsAsList() {
+    return instants;
```

Comment:
should have a clean up ticket to call this `getInstants` and the other `getInstantsAsStream`. Public API names should not cause confusion

Reply:
agree.